registry/application/services/organization_publisher_service.py

```diff
@@ -219,4 +225,16 @@ def update_verification(self, verification_type, verification_details):
         else:
             logger.error(f"Invalid verification type {verification_type}")
             raise MethodNotImplemented()
+
+        # TODO send_email should not have boto_utils
+        try:
+            if organization:
+                contacts = []
+                for contact in organization.contacts:
```

Comment:
Will contact['email_id'] be a valid email always? If any one is invalid will the other emails be sent or will all fail?

Reply:
mail will go for valid emails and fail for invalid ones.